code/modules/projectiles/guns/projectile/pistol/colt.dm

```diff
@@ -12,7 +12,7 @@
 	can_dual = TRUE
 	load_method = SINGLE_CASING|MAGAZINE
 	mag_well = MAG_WELL_PISTOL | MAG_WELL_H_PISTOL
-	damage_multiplier = 1.2
+	damage_multiplier = 1.1
```

Comment:
See below. The Delta Elite is worse  than the Colt base. Why make the Blackshield have a gun with more recoil and less damage - yet make it cost way more.

Reply:
Delta is 10mm this is 9mm
So Delta should be higher base with bullet but does techinnally lose to 9mm

Reply:
Shit - neglected that. I'll fix that tomorrow. I completely forgot we're stupid and made the Colt 9mm instead of a round more akin to what it would actually be.